### DISCUSSION TITLE
Limit Language Options by Character Sheet

### DISCUSSION THREAD

**Discussion by AndrewRP8023** [mclemente/fvtt-module-polyglot#286] [Ideas]
Would it be possible to add a Yes/No option to the settings to limit the languages in the chat drop-down to just what's on a player's character sheet? So if they only speak Common, Elvish, and Gnomish, then the drop-down would only display those three languages as options for them to speak?

**Comment by mclemente** [comment]
That's already a thing. Do the players have a character assigned to them?

Unless you mean to limit the language options for the GM based on the languages the PCs know.
